### PR TITLE
Add support for specifying a minimum Xcode version.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

File: bazel.sh [hunk 1]
```diff
@@ -22,6 +22,7 @@
 # Arguments:
 #   1. bazel action (build or test, usually)
 #   2. BUILD target.
+#   3. Minimum Xcode version. E.g. "8" or "8.2.1"
```
Comment (ianegordon) [inline bazel.sh:25]: Does it make sense to have a default value equal to what is locally installed?
xcrun xcodebuild -version

Comment (jverkoey) [reply bazel.sh:25]: By default it'll run against every Xcode installation available, which I think may be preferable for this script. What do you think?

Comment (ianegordon) [reply bazel.sh:25]: SGTM.

**Comment by jverkoey** [comment]
Can haz lgtm? :)

**Review by ajsecord — APPROVED** [review]

**Pull request merged**